secrets-provider-test-chart/values.yaml

```diff
@@ -0,0 +1,25 @@
+global:
+  namespace: local-secrets-provider
+
+# K8s permission defaults
+rbac:
+  roleName: secrets-provider-role
+  roleBinding: secrets-provider-role-binding
+  serviceAccount:
+    # Specifies whether a service account should be created
+    create: true
+    # The name of the service account to use.
+    # If not set and create is true, a name is generated using the fullname template
+    name: cucumber
+
+secretsProvider:
+  image: cyberark/secrets-provider-for-k8s
+  tag: 1.0.0
+  name: cyberark-secrets-provider
+
+secretsProviderEnvironment:
+  conjurSslCertificateValue: ""
+  conjurSslCertificateName: secrets-provider-ssl-config-map
+  secretsDestination: k8s_secrets
```

Comment:
What other options we have here?
Since we currently do not support any other than `k8s_secrets`, we shouldn't supply it as variable.